### PR TITLE
Add navigateTo helper method to make sure we always use right nav method

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: extension/src/popup/helpers/navigateTo.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { history } from "popup/constants/history";
+
+export const navigateTo = (pathname: string) => history.push({ pathname });
```
Comment (piyalbasu) [inline extension/src/popup/helpers/navigateTo.ts:3]: I really wanted this to accept pathname as type `ROUTES` but we have some instances where we nav using `${ROUTES}?${queryParam}` which ruined my well laid plans

Doesn't appear to be a way to create a type to handle both

Comment (morleyzhi) [reply extension/src/popup/helpers/navigateTo.ts:3]: What if `navigateTo` took a second optional param `queryParams`?

**Review by morleyzhi — APPROVED** [review]

File: extension/src/popup/helpers/navigateTo.ts (hunk 1 above)
Comment (vcarl) [reply extension/src/popup/helpers/navigateTo.ts:3]: Or maybe `route | LocationObject`?

**Review by jeesunikim — APPROVED** [review]
merci!

**Pull request merged**